### DISCUSSION TITLE
Access to private Projects for Outside Collaborators

### DISCUSSION THREAD

**Discussion by dbabokin** [community/community#7562] [Projects and Issues] [answered]
I've created a private Project in Organization scope. Now I'm trying to give access for more people. I can do that for Organization "Members", but not "Outside Collaborators", though it seems to be logical to be able to give at lease read access to Outside Collaborators as well.

**Comment by azenMatt** [comment]
Hi @dbabokin - we are working on this at the moment - let me know if you would like your organization to be added to the feature flag - just let me know the org name 😄

**Reply by scherii** [reply]
We‘d love to opt-in with Webgem!

https://github.com/Webgem/

**Reply by dbabokin** [reply]
Yeah, please enroll https://github.com/ispc
Thank you!

**Reply by azenMatt** [reply]
Hi @scherii @dbabokin  - I have added both of those orgs 

As a heads up - this list of possible outside collaborators is anyone who is already a collaborator on a repo. Let me know how it goes!

**Reply by scherii** [reply]
Hey @azenMatt 

I have just tested it, it works perfectly.

However, we can't use it for our purposes because we don't want to give the outside collaborators access to the code in the private repository. For us it would be ideal if we could give users access to the project management regardless of repository access.

**Reply by azenMatt** [reply]
thanks for the feedback - that is a bigger level of permission changes that we need there, but I understand where you are coming from and this is a common request that we will continue to look into. No timeline yet for when this will be resolved, but it is something I would expect us to have a new permission for in the future

**Comment by azenMatt** [comment, marked as answer]
This functionality is now live! 

More details in this weeks changelog here - https://github.blog/changelog/2022-02-02-the-new-github-issues-02-02-update/
